client/dashboard/components/icon-list/icon-list-item.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import clsx from 'clsx';
+import { forwardRef } from 'react';
+import type { IconListItemProps } from './types';
+
+import './icon-list-item.scss';
+
+function UnforwardedIconListItem(
+	{ title, description, decoration, suffix, className }: IconListItemProps,
+	ref: React.ForwardedRef< HTMLSpanElement >
+) {
+	return (
+		<VStack className={ clsx( 'icon-list-item', className ) } ref={ ref } as="span">
+			<HStack spacing={ 3 } justify="flex-start" alignment="center" as="span">
```

Comment:
```suggestion
			<HStack spacing={ 4 } justify="flex-start" alignment="center" as="span">
```

If I'm not mistaken, based on the designs, the gap should be `16px`.

Reply:
Good catch